Oral health updates to variables/variable-details

caitlink12 opened the pull request:
Oral health variables have been added to both variable and variables details along with minor updates to variable details.

yulric commented on inst/extdata/variable_details.csv:
```diff
@@ -4,11 +4,11 @@ active_transport,N/A,cont,"cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs201
-ADL_01,ADL_01_cat2_NA::a,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p:: ...
-ADL_01,ADL_01_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p:: ...
-ADL_01,ADL_01_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p:: ...
+ADL_01,ADL_01_cat2_1,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A, cchs ...
+ADL_01,ADL_01_cat2_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A, cchs ...
+ADL_01,ADL_01_cat2_NA::a,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A,  ...
+ADL_01,ADL_01_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A,  ...
+ADL_01,ADL_01_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_p::RACA_6A, cchs2003_p::RACC_6A, cchs2005_p::RACE_6A, cchs2007_2008_p::RAC_6A,  ...
```
@caitlink12 Can you explain why you removed the PUMF 2015-2016 databases from these rows?

caitlink12 replied:
I believe this was a typo Doug had pointed out previously that he wanted me to take a look into. The 2015-16 ADL_01 variable was reformatted to a 4 category variable in these survey cycles so their inclusion in these rows would be incorrect. Additional rows had been added to variable_details to account for their harmonization back to a two category variable as well as rows to have them coded as their original 4-category variable. There also should not be 2015-2016 or 2017-18 master cycles for these variables as the module wasn't mandatory and therefore not collected in Ontario for those cycles.

yulric commented on inst/extdata/variable_details.csv:
```diff
@@ -111,19 +111,19 @@ ADL_07,ADL_07_cat2_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2001_i, cchs20
-ADL_der,ADL_der_cat2_1,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,1,2,Needs help with tasks,Needs help with tasks,N/A,N/A,Needs help with tasks,Derived help tasks,Deri ...
-ADL_der,ADL_der_cat2_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,2,2,Does not need help with tasks,Does need help with tasks,N/A,N/A,Does need help with tasks,Derived ...
-ADL_der,ADL_der_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,NA::b,2,missing,missing,N/A,N/A,Does need help with tasks,Derived help tasks,Derived needs help w ...
+ADL_der,ADL_der_catN/A_Func::calculate_adl,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03,  ...
+ADL_der,ADL_der_cat2_1,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/ ...
+ADL_der,ADL_der_cat2_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/ ...
+ADL_der,ADL_der_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]" ...
```
@caitlink12 Instead of using the suffix `_i`, can you use the `_m` suffix? I talked to Stacey and the plan is to change all the `_s`, `_m`, `_i` to `_m`.

caitlink12 replied:
Absolutely, I will make these changes.

yulric commented on inst/extdata/variable_details.csv:
```diff
@@ -111,19 +111,19 @@ ADL_07,ADL_07_cat2_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2001_i, cchs20
-ADL_score_5,ADL_score_5_cat6_6,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,5,6,Needs help with at least 5 tasks,Needs help with at least 5 tasks,N/A,N/A,Needs help with ...
-ADL_score_5,ADL_score_5_cat6_NA::a,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,NA::a,6,not applicable,not applicable,N/A,N/A,not applicable,ADL score,"Derived using the ...
-ADL_score_5,ADL_score_5_cat6_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,NA::b,6,missing,missing,N/A,N/A,missing,ADL score,"Derived using the ADL variables common ...
+ADL_score_5,ADL_score_5_cat6_1,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_ ...
+ADL_score_5,ADL_score_5_cat6_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_ ...
+ADL_score_5,ADL_score_5_cat6_3,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_ ...
+ADL_score_5,ADL_score_5_cat6_4,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_ ...
+ADL_score_5,ADL_score_5_cat6_5,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_ ...
+ADL_score_5,ADL_score_5_cat6_6,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_ ...
+ADL_score_5,ADL_score_5_cat6_NA::a,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04,  ...
+ADL_score_5,ADL_score_5_cat6_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04,  ...
```
@caitlink12 Same as [this comment](https://github.com/Big-Life-Lab/cchsflow/pull/140/commits/db122101b4616be897bba7504d1a33352759da97#r2291391462).

yulric commented on inst/extdata/variable_details.csv:
```diff
@@ -510,11 +510,11 @@ CCC_17A,CCC_17A_cat5_NA::a,cat,"cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cc
-CCC_181,CCC_181_cat2_NA::a,cat,"cchs2009_s, cchs2010_s, cchs2012_s",[CCC_181],,cat,NA::a,2,not applicable,not applicable,N/A,6,not applicable,Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,
-CCC_181,CCC_181_cat2_NA::b,cat,"cchs2009_s, cchs2010_s, cchs2012_s",[CCC_181],,cat,NA::b,2,missing,missing,N/A,"[7,9]",don't know (7); refusal (8); not stated (9),Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,
-CCC_181,CCC_181_cat2_NA::b,cat,"cchs2009_s, cchs2010_s, cchs2012_s",[CCC_181],,cat,NA::b,2,else,else,N/A,else,else,Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,
+CCC_181,CCC_181_cat2_1,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,1,2,Yes alzheimer's disease or other dementia,Yes alzheime ...
+CCC_181,CCC_181_cat2_2,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,2,2,No alzheimer's disease or other dementia,No alzheimer's ...
+CCC_181,CCC_181_cat2_NA::a,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,NA::a,2,not applicable,not applicable,N/A,6,not applica ...
+CCC_181,CCC_181_cat2_NA::b,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,NA::b,2,missing,missing,N/A,"[7,9]",don't know (7); ref ...
+CCC_181,CCC_181_cat2_NA::b,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,NA::b,2,else,else,N/A,else,else,Alzheimer's disease or  ...
```
@caitlink12 Can you do the same as in [this comment](https://github.com/Big-Life-Lab/cchsflow/pull/140/commits/db122101b4616be897bba7504d1a33352759da97#r2291391462)

yulric commented on inst/extdata/variable_details.csv:
```diff
@@ -561,6 +561,16 @@ COPD_Emph_der,COPD_Emph_der_cat3_1,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs
 COPD_Emph_der,COPD_Emph_der_cat3_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[DHHGAGE_cont, CCC_91E, CCC_91F, CCC_091]",,N/A,2,3,Under 35 years with condition,Age under 35 years with condition,N/A,N/A,Respiratory  ...
 COPD_Emph_der,COPD_Emph_der_cat3_3,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[DHHGAGE_cont, CCC_91E, CCC_91F, CCC_091]",,N/A,3,3,No condition,No condition,N/A,N/A,Respiratory Condition,COPD/Emphysema,COPD/Emphysem ...
 COPD_Emph_der,COPD_Emph_der_cat3_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[DHHGAGE_cont, CCC_91E, CCC_91F, CCC_091]",,N/A,NA::b,3,missing,missing,N/A,N/A,Respiratory Condition,COPD/Emphysema,COPD/Emphysema, ...
+DEN_132,DEN_132_cat7_1,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,1,7,< 1 year,< 1 year,N/A,1,< 1 year,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_2,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,2,7,1 to < 2 years,1 to < 2 years,N/A,2,1 to < 2 years,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_3,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,3,7,2 to < 3 years,2 to < 3 years,N/A,3,2 to < 3 years,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_4,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,4,7,3 to < 4 years,3 to < 4 years,N/A,4,3 to < 4 years,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_5,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,5,7,4 to < 5 years,4 to < 5 years,N/A,5,4 to < 5 years,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_6,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,6,7,5 years or more,5 years or more,N/A,6,5 years or more,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_7,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,7,7,Never,Never,N/A,7,Never,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_NA::a,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,NA::a,7,not applicable,not applicable,N/A,96,not applicable,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_NA::b,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,NA::b,7,missing,missing,N/A,"[97,99]",don't know (97); refusal (98); not stated (99),Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
+DEN_132,DEN_132_cat7_NA::b,cat,"cchs2001_i, cchs2003_i, cchs2005_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","cchs2001_i::DENA_132, cchs2003_i::DENC_132, cchs2005_i::DENE_132, [DEN_132]",ICES,cat,NA::b,7,missing,missing,N/A,else,else,Visited dental professional - last time,Visited dental professional - last time,,3.0.0,lastUpdate,active,,
```
@caitlink12 Can you do the same as in [this comment](https://github.com/Big-Life-Lab/cchsflow/pull/140/commits/db122101b4616be897bba7504d1a33352759da97#r2291391462)

yulric commented on inst/extdata/variable_details.csv:
```diff
@@ -2160,6 +2170,35 @@ number_conditions,number_conditions_cat6_2,cat,"cchs2001_p, cchs2003_p, cchs2005
 number_conditions,number_conditions_cat6_3,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[CCC_121, CCC_131, CCC_151, CCC_171, CCC_280, resp_condition_der, CCC_051]",,N/A,3,6,3,3,N/A,N/A,N/A,Number of conditions,Number of chronic conditions,,3.0.0,last ...
 number_conditions,number_conditions_cat6_4,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[CCC_121, CCC_131, CCC_151, CCC_171, CCC_280, resp_condition_der, CCC_051]",,N/A,4,6,4,4,N/A,N/A,N/A,Number of conditions,Number of chronic conditions,,3.0.0,last ...
 number_conditions,number_conditions_cat6_5+,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s, cchs2010_s, cchs2012_s","DerivedVar::[CCC_121, CCC_131, CCC_151, CCC_171, CCC_280, resp_condition_der, CCC_051]",,N/A,5,6,5+,5+,N/A,N/A,N/A,Number of conditions,Number of chronic conditions,,3.0.0,l ...
+OH1_20,OH1_20_cat5_1,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_20, [OH1_20]",ICES,cat,1,5,Excellent,Excellent,N/A,1,Excellent,Self-percieved oral health,Self-percieved health of teeth and mouth,,3.0.0,lastUpdated,active,,
+OH1_20,OH1_20_cat5_2,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_20, [OH1_20]",ICES,cat,2,5,Very good,Very good,N/A,2,Very good,Self-percieved oral health,Self-percieved health of teeth and mouth,,3.0.0,lastUpdated,active,,
+OH1_20,OH1_20_cat5_3,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_20, [OH1_20]",ICES,cat,3,5,Good,Good,N/A,3,Good,Self-percieved oral health,Self-percieved health of teeth and mouth,,3.0.0,lastUpdated,active,,
+OH1_20,OH1_20_cat5_4,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_20, [OH1_20]",ICES,cat,4,5,Fair,Fair,N/A,4,Fair,Self-percieved oral health,Self-percieved health of teeth and mouth,,3.0.0,lastUpdated,active,,
+OH1_20,OH1_20_cat5_5,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_20, [OH1_20]",ICES,cat,5,5,Poor,Poor,N/A,5,Poor,Self-percieved oral health,Self-percieved health of teeth and mouth,,3.0.0,lastUpdated,active,,
+OH1_20,OH1_20_cat5_NA::a,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_20, [OH1_20]",ICES,cat,NA::a,5,not applicable,not applicable,N/A,6,not applicable,Self-percieved oral health,Self-percieved health of teeth and mouth,,3.0.0,lastUpdated,active,,
+OH1_20,OH1_20_cat5_NA::b,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_20, [OH1_20]",ICES,cat,NA::b,5,missing,missing,N/A,"[7,9]",don't know (7); refusal (8); not stated (9),Self-percieved oral health,Self-percieved health of teeth and mouth,,3.0.0,lastUpdated,active,,
+OH1_20,OH1_20_cat5_NA::b,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_20, [OH1_20]",ICES,cat,NA::b,5,missing,missing,N/A,else,else,Self-percieved oral health,Self-percieved health of teeth and mouth,,3.0.0,lastUpdated,active,,
+OH1_22,OH1_22_cat4_1,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_22, [OH1_22]",ICES,cat,1,4,Often,Often,N/A,1,Often,Freq. of teeth/gum pain - last month,Frequency of pain in teeth/gums - past month,,3.0.0,lastUpdated,active,,
+OH1_22,OH1_22_cat4_2,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_22, [OH1_22]",ICES,cat,2,4,Sometimes,Sometimes,N/A,2,Sometimes,Freq. of teeth/gum pain - last month,Frequency of pain in teeth/gums - past month,,3.0.0,lastUpdated,active,,
+OH1_22,OH1_22_cat4_3,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_22, [OH1_22]",ICES,cat,3,4,Rarely,Rarely,N/A,3,Rarely,Freq. of teeth/gum pain - last month,Frequency of pain in teeth/gums - past month,,3.0.0,lastUpdated,active,,
+OH1_22,OH1_22_cat4_4,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_22, [OH1_22]",ICES,cat,4,4,Never,Never,N/A,4,Never,Freq. of teeth/gum pain - last month,Frequency of pain in teeth/gums - past month,,3.0.0,lastUpdated,active,,
+OH1_22,OH1_22_cat4_NA::a,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_22, [OH1_22]",ICES,cat,NA::a,4,not applicable,not applicable,N/A,6,not applicable,Freq. of teeth/gum pain - last month,Frequency of pain in teeth/gums - past month,,3.0.0,lastUpdated,active,,
+OH1_22,OH1_22_cat4_NA::b,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_22, [OH1_22]",ICES,cat,NA::b,4,missing,missing,N/A,"[7,9]",don't know (7); refusal (8); not stated (9),Freq. of teeth/gum pain - last month,Frequency of pain in teeth/gums - past month,,3.0.0,lastUpdated,active,,
+OH1_22,OH1_22_cat4_NA::b,cat,"cchs2003_i, cchs2007_2008_i, cchs2011_2012_i, cchs2013_2014_i","cchs2003_i::OH1C_22, [OH1_22]",ICES,cat,NA::b,4,missing,missing,N/A,else,else,Freq. of teeth/gum pain - last month,Frequency of pain in teeth/gums - past month,,3.0.0,lastUpdated,active,,
+OH2_20,OH2_20_cat2_1,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_20, cchs2005_i::OH2E_20, cchs2017_2018_i::OHT_030, [OH2_20]",ICES,cat,1,2,Yes,Yes,N/A,1,Yes,Has one or more of own teeth,Has one or more of own teeth,,3.0.0,lastUpdated,active,,
+OH2_20,OH2_20_cat2_2,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_20, cchs2005_i::OH2E_20, cchs2017_2018_i::OHT_030, [OH2_20]",ICES,cat,2,2,No,No,N/A,2,No,Has one or more of own teeth,Has one or more of own teeth,,3.0.0,lastUpdated,active,,
+OH2_20,OH2_20_cat2_1NA::a,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_20, cchs2005_i::OH2E_20, cchs2017_2018_i::OHT_030, [OH2_20]",ICES,cat,NA::a,2,not applicable,not applicable,N/A,6,not applicable,Has one or more of own teeth,Has one or more of own teeth,,3.0.0,lastUpdated,active,,
+OH2_20,OH2_20_cat2_NA::b,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_20, cchs2005_i::OH2E_20, cchs2017_2018_i::OHT_030, [OH2_20]",ICES,cat,NA::b,2,missing,missing,N/A,"[7,9]",don't know (7); refusal (8); not stated (9),Has one or more of own teeth,Has one or more of own teeth,,3.0.0,lastUpdated,active,,
+OH2_20,OH2_20_cat2_NA::b,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_20, cchs2005_i::OH2E_20, cchs2017_2018_i::OHT_030, [OH2_20]",ICES,cat,NA::b,2,missing,missing,N/A,else,else,Has one or more of own teeth,Has one or more of own teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_1,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,1,6,> Twice/Day,> Twice/Day,N/A,1,> Twice/Day,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_2,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,2,6,Twice/Day,Twice/Day,N/A,2,Twice/Day,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_3,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,3,6,Once/Day,Once/Day,N/A,3,Once/Day,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_4,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,4,6,< Once/Day,< Once/Day,N/A,4,< Once/Day,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_5,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,5,6,Once/Week,Once/Week,N/A,5,Once/Week,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_6,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,6,6,< Once/Week,< Once/Week,N/A,6,< Once/Week,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_NA::a,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,NA::a,6,not applicable,not applicable,N/A,96,not applicable,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_NA::b,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,NA::b,6,missing,missing,N/A,"[97,99]",don't know (97); refusal (98); not stated (99),Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
+OH2_30,OH2_30_cat6_NA::b,cat,"cchs2003_i, cchs2005_i, cchs2009_20010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2017_2018_i","cchs2003_i::OH2C_30, cchs2005_i::OH2E_30, cchs2017_2018_i::DEN_101, [OH2_30]",ICES,cat,NA::b,6,missing,missing,N/A,else,else,Freq. brushing teeth,Frequency of brushing teeth,,3.0.0,lastUpdated,active,,
```
@caitlink12 Can you do the same as in [this comment](https://github.com/Big-Life-Lab/cchsflow/pull/140/commits/db122101b4616be897bba7504d1a33352759da97#r2291391462)

Commit:
fixed bug with ADL_01 variable

I believe this was a typo Doug had pointed out previously that he wanted me to
take a look into. The 2015-16 ADL_01 variable was reformatted to a 4 category
variable in these survey cycles so their inclusion in these rows would be
incorrect. Additional rows had been added to variable_details to account for
their harmonization back to a two category variable as well as rows to have
them coded as their original 4-category variable. There also should not be
2015-2016 or 2017-18 master cycles for these variables as the module wasn't
mandatory and therefore not collected in Ontario for those cycles.

Commit:
updated ADL_der variable

Commit:
updated CCC_181 variable

Commit:
new harmonized variable, last time visited dental professional

Commit:
harmonzied new variable, self-perceived oral health

Commit:
harmonized new variable, frequency teeth/gum pain

Commit:
harmonized new variable, has one or more of own teeth

Commit:
harmonized new variable, frequency of brushing teeth

yulric commented on inst/extdata/variable_details.csv:
```diff
@@ -2849,10 +2888,10 @@ SMK_208,N/A,cont,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_
 SMK_208,N/A,cont,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i, cchs2019_2020_i, cchs2021_i, cchs ...
 SMK_208,N/A,cont,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i, cchs2019_2020_i, cchs2021_i, cchs ...
 SMK_208,N/A,cont,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i, cchs2019_2020_i, cchs2021_i, cchs ...
-SMKG01C_cont,SMKG01C_cont_copy_cchs2009_p_8,cont,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i, cchs2019_2020_i, cchs2021_i, cchs2022_i, cchs2023_i","cchs2001_i::SMKA_01C, cchs2003_i::SMKC_01C, cchs2005_i::SMKE_01C, cchs20_i::SMKA_01C, cchs2015_2016_i::SMK_035, cchs2017_ ...
-SMKG01C_cont,SMKG01C_cont_NAa_cchs2009_p,cont,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i, cchs2019_2020_i, cchs2021_i, cchs2022_i, cchs2023_i","cchs2001_i::SMKA_01C, cchs2003_i::SMKC_01C, cchs2005_i::SMKE_01C, cchs20_i::SMKA_01C, cchs2015_2016_i::SMK_035, cchs2017_201 ...
-SMKG01C_cont,SMKG01C_cont_NAb_cchs2009_p_9,cont,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i, cchs2019_2020_i, cchs2021_i, cchs2022_i, cchs2023_i","cchs2001_i::SMKA_01C, cchs2003_i::SMKC_01C, cchs2005_i::SMKE_01C, cchs20_i::SMKA_01C, cchs2015_2016_i::SMK_035, cchs2017_2 ...
-SMKG01C_cont,SMKG01C_cont_NAb_cchs2009_p_e,cont,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i, cchs2019_2020_i, cchs2021_i, cchs2022_i, cchs2023_i","cchs2001_i::SMKA_01C, cchs2003_i::SMKC_01C, cchs2005_i::SMKE_01C, cchs20_i::SMKA_01C, cchs2015_2016_i::SMK_035, cchs2017_2 ...
```
@caitlink12 Was this change done by mistake? Did you mean to remove the rows for SMKG01C_cont variables?

caitlink12 replied:
I'm not sure, this may have been a change Miakol made when she was going through the smoking variables for Doug? If possible maybe put the rows back in and I can have a quick look at what was deleted because right now it appears the sheet I have still has some version of the deleted row still included

yulric replied:
I took a look at the diff and the removed rows are in the file, they've just been moved lower. In addition, it looks like the `cchs2001_i` database was removed from those rows but it is present in other `SMKG01C_cont` rows. Does that make sense?

caitlink12 replied:
Okay if that's the case it's likely that the variable categories that were harmonized into continuous values were different. So I likely made a mistake about the category values for 2001_i and had to correct the rows it showed up in.

Commit:
updates to the SMK_01B variable details. added row for it to variables sheet

Commit:
updates to SMKG01C_A variable

Commit:
updated ADL_score_5 variable

Commit:
updates to the SMKG01C_B variable

Commit:
updates to the SMKG01C_cont variable
